requirements/edx/base.txt

```diff
@@ -55,8 +54,9 @@ Markdown==2.2.1
 --allow-unverified meliae
 meliae==0.4.0
 mongoengine==0.7.10
+MySQL-python==1.2.5
 networkx==1.7
-nltk==2.0.4
+nltk>=2.0.4 # upgrade to 2.0.5 once edx-ora2 is compatible
```

Comment:
In general, I'd rather see the library (edx-ora2) with flexible versions, and the application (edx-platform) with fixed ones.